docs/userguide.md

```diff
@@ -79,18 +78,18 @@ sudo mv kubefedctl /usr/local/bin/ # make sure the location is in the PATH
 
 **NOTE:** `kubefedctl` is built for Linux only in the release package.
```

Comment:
This is no longer true - we now include a darwin binary for osx with each release.

Reply:
Added OSX to the note - let me know if you want me to remove it instead.